### PR TITLE
Abs function addition

### PR DESCRIPTION
**Pull request by jiachenwang**
Added Abs function for decimal following the Postgres standard. Added 2 unit tests - one to check working of the Abs function internally and one with SQL.

### CODE REVIEW

**Comment by hzxa21** [comment]
@jiachenwang It looks like you are failing a test case. Can you fix that and request a review?

**Comment by jiachenwang** [comment]
Sure, I will fix it and request again ASAP.

File: src/codegen/query_compiler.cpp [hunk 1]
```diff
@@ -119,7 +119,7 @@ bool QueryCompiler::IsExpressionSupported(
     const expression::AbstractExpression &expr) {
   switch (expr.GetExpressionType()) {
     case ExpressionType::STAR:
-    case ExpressionType::FUNCTION:
+    // case ExpressionType::FUNCTION:
```
Comment (pmenon) [inline src/codegen/query_compiler.cpp]: Remove this. We'll fix when we have function parity with existing value-based functions.

**Comment by apavlo** [comment]
Replaced by #1031

**Pull request closed without merging**